src/components/Checkbox/tests/Checkbox.test.tsx

```diff
@@ -23,6 +24,18 @@ describe('<Checkbox />', () => {
       element.find('input').simulate('change');
       expect(spy).toHaveBeenCalledWith(true, 'MyCheckbox');
     });
+
+    // JSDOM is overwriting currentTarget
+    // See https://github.com/airbnb/enzyme/issues/218
+    // eslint-disable-next-line jest/no-disabled-tests
+    it.skip('renders when the correct focus when checkbox is toggled off', () => {
```

Comment:
```suggestion
     it.skip('renders with the correct focus when checkbox is toggled off', () => {
```

Reply:
I don't fully understand why this test is skipped.

Reply:
There is an issue with enzyme and JSDom where JSDom is overwritting `currentTarget` on events. It seems that since I created this pr the maintainer of enzyme has stated this probably won't be fixed since he seems to have a personal grudge against `simulate`. So I removed the skip and added a hacky solution to get the test to pass 🤷‍♂️